tests/src/test_cases/widgets/test_calendar.c

```diff
@@ -220,4 +220,20 @@ void test_calendar_custom_year_list(void)
     TEST_ASSERT_EQUAL_SCREENSHOT("widgets/calendar_08.png");
 }
 
+void test_calendar_lunar(void)
+{
+    lv_obj_clean(lv_screen_active());
```

Comment:
Not needed

Reply:
If this line is not present, the image from the previous example will be retained here

Reply:
Sorry for the confusion. Please use the `g_calendar` object. The existing test case above this should use it but doesn't.

```c
void test_calendar_lunar(void)
{
    lv_obj_set_size(g_calendar, 400, 350);
    lv_obj_center(g_calendar);
    lv_calendar_set_today_date(g_calendar, 2024, 03, 22);
    lv_calendar_set_showed_date(g_calendar, 2024, 03);

    lv_obj_set_style_text_font(g_calendar, &lv_font_simsun_14_cjk, LV_PART_MAIN);
    lv_calendar_set_lunar_mode(g_calendar, true);

    TEST_ASSERT_EQUAL_SCREENSHOT("widgets/calendar_09.png");
}
```

Reply:
Thank you